app/models/task.rb

```diff
@@ -294,6 +329,10 @@ def trigger_options
       triggers.map { |t, _| [t.gsub('on_', '').titlecase, t] }
     end
 
+
```

Comment:
In general, we try to clean up unnecessary comments and space before accepting a PR onto master. I know this isn't necessarily at that state yet, but just a heads up.